vars/runATH.groovy

```diff
@@ -120,56 +125,81 @@ def call(Map params = [:]) {
             }
 
             def testingbranches = ["failFast": failFast]
-            for (browser in browsers) {
-                if (supportedBrowsers.contains(browser)) {
-
-                    def currentBrowser = browser
+            for (jdk in jdks) {
+                if (supportedJdks.contains(jdk)) {
+                    def currentJdk = jdk
+                    def javaOptions = defaultJavaOptions.clone()
+                    def commandBaseWithFutureJava = ""
                     def containerArgs = "-v /var/run/docker.sock:/var/run/docker.sock -u ath-user"
+
                     if(configFile) {
                         containerArgs += " -e CONFIG=../${configFile}" // ATH runs are executed in a subfolder, hence path needs to take that into account
                     }
-                    def commandBase = "./run.sh ${currentBrowser} ./jenkins.war -B -Dmaven.test.failure.ignore=true -DforkCount=1 -B -Dsurefire.rerunFailingTestsCount=${rerunCount}"
 
-                    if (testsToRun) {
-                        testingbranches["ATH individual tests-${currentBrowser}"] = {
-                            dir("test${currentBrowser}") {
-                                def discriminator = "-Dtest=${testsToRun}"
-                                test(discriminator, commandBase, localSnapshots, localPluginsStashName, containerArgs, athContainerImage, javaOptions)
-                            }
+                    // Add options for jdks
+                    if ( currentJdk  > 8) {
```

Comment:
Can be squashed into a single `if` after the removal of Java 10 support

Reply:
Done, I left it so just in case some time in the future we add java 9 or 10, but with the current support versions, it's fine as you say.

Reply:
I would rather expect Java 12 :)